BlockSettleUILib/Trading/RFQTicketXBT.cpp

```diff
@@ -336,6 +336,35 @@ void RFQTicketXBT::fillRecvAddresses()
    }
 }
 
+bool RFQTicketXBT::preSubmitCheck()
+{
+   if (currentGroupType_ == ProductGroupType::XBTGroupType) {
+      const auto qty = getQuantity();
+      const auto& tradeSettings = authAddressManager_->tradeSettings();
+      assert(tradeSettings);
+
+      bool validAmount = false;
+      if (currentProduct_ == UiUtils::XbtCurrency) {
+         validAmount = tradeSettings->xbtTier1Limit > bs::XBTAmount(qty).GetValue();
+      } else {
+         const double indPrice = getIndicativePrice();
+         bs::XBTAmount price(indPrice * (1 + (tradeSettings->xbtPriceBand / 100)));
+         validAmount = price > bs::XBTAmount(qty);
+      }
+
+      if (!validAmount) {
+         BSMessageBox(BSMessageBox::info
+            , tr("Notice"), tr("Authentication Address not verified")
+            , tr("Trades above 1 XBT are not permitted for non-verified Authentication Addresses. "
```

Comment:
please do not use hardcoded amount. There is exact value in trading settings which you are using for validation.

Reply:
fixed, thanks